devicon.json

```diff
@@ -6654,6 +6654,30 @@
         "color": "#dd4814",
         "aliases": []
     },
+    {
+        "name": "unifiedmodelinglanguage",
+        "altnames": [
+            "uml",
+            "unified modeling language"
+        ],
+        "tags": [
+            "modeling",
+            "design",
+            "language"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark"
+            ],
+            "font": [
+                "plain",
+                "plain-wordmark"
+            ]
+        },
+        "color": "#452e7f",
```

Comment:
@Snailedlt The color of UML is here ahah

Reply:
Lol, mb!
Must have been because I checked the ordering, and forgot to scroll back to where I started 😅

The color is still not the same as the logo on their website though: https://uml.org

Reply:
![image](https://user-images.githubusercontent.com/107955904/182458223-a87502b5-f331-461a-97aa-18dd0d6df590.png)

It's the purple color part, isn't it ?

Reply:
I disagree, the purple is exactly that color : 

![image](https://user-images.githubusercontent.com/107955904/183243838-376e733b-71b4-4330-80a4-f286e91f9deb.png)

Don't you have the same ?

Reply:
If you're using the windows powertool to check, then that explains it. That seems to be wrong some times.

You can see the exact svg they use in my earlier comment:
https://github.com/devicons/devicon/pull/1241#discussion_r934551790